release-notes/release-notes53.md

```diff
@@ -41,9 +80,18 @@ For information on the PowerPaste plugin, see: [PowerPaste plugin]({{site.baseur
 
 The {{site.productname}} 5.3 release includes an accompanying release of the **Spellchecker Pro** premium plugin.
 
-**Spellchecker Pro** 2.0.2
+**Spellchecker Pro** 2.0.2 
```

Comment:
This should probably be `Spell Checker Pro`, as that's what it is elsewhere on the site.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1482/commits/a371f8e8bf84dbb8784ef55e828d1af6ddb5b69f